From 0b3f125bec8678dd2e2cd19c3ebba35f51e06a3f Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 13 Aug 2024 15:43:45 +0200
Subject: [PATCH] arm64: defconfig: addition of DCMI in fragment-02

Addition of the STM32 DCMI driver within
fragment-02-defconfig-addons.config

Change-Id: I66aae474d4114f73b93f7bbec726c18438010f63
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/398242
Domain-Review: Stephane DANIEAU <stephane.danieau@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Stephane DANIEAU <stephane.danieau@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -117,5 +117,6 @@ CONFIG_USB_STM32_USBH=y
 CONFIG_V4L_TEST_DRIVERS=y
 CONFIG_VIDEO_IMX335=m
 CONFIG_VIDEO_STM32_CSI=m
+CONFIG_VIDEO_STM32_DCMI=m
 CONFIG_VIDEO_STM32_DCMIPP=m
 CONFIG_VIDEO_VIVID=m
